environment/bashrc/.bashrc_linux64

```diff
@@ -46,10 +46,10 @@ case $target in
     if [[ -d $HOME/privatemodules ]]; then
       module use --append $HOME/privatemodules
     fi
-    dm_core="eospac git tk ndi python doxygen numdiff totalview \
-dia graphviz ack"
-    dm_gcc="gcc/7.2.0 cmake netlib-lapack gsl metis random123 csk qt"
-    dm_openmpi="openmpi parmetis superlu-dist trilinos valgrind"
+    dm_core="ack dia doxygen git graphviz htop mscgen numdiff python random123 \
+tk ndi totalview"
+    dm_gcc="gcc cmake eospac/6.3.1_r20161202150449 gsl metis netlib-lapack qt"
```

Comment:
@KineticTheory is 6.3.1 correct? It's 6.3.0 above

Reply:
BTW - eospac/6.3.1 has the same API/ABI as eospac/6.3.0 so both should work with the current version of our codes.